about.html

```diff
@@ -0,0 +1,121 @@
+			<ul class="nav-menu">
+				<li class="about"><a href="about.html">About</a></li>
+				<li><a href="index.html">Home</a></li>
+				<li><a href="#">Program</a></li>
+				<li><a href="#">Join</a></li>
+				<li><a href="#">Sponsors</a></li>
+				<li><a href="#">News</a></li>
+				<li id="camp"><a href="#">Campaign</a></li>
+			</ul>
+		</div>
+	</section>
+	<div class="desktop-p">
+		<p class="moto">Keeping You Healthy</p>
+		<h1> RAY'S ELEMENT AQUA <br /> GLOBAL SUMMIT 2021 </h1>
+	</div>
+	<p class="desktop-article">Element Aqua Summit is the most highly influential and profitable network for
+		Carribean's health,fitness and wellness industry.
+		we organize health, fitness and wellness conferences, events that moves even the most fable minds to
+		achieve their highest potential as far as living healthy is concerned. Our Awards duely recognise
+		leading companies and excutives from accross the health, fitness and wellness ecosystem.
+	</p>
+	<div class="article2">
+		<p>Please reach us through email for any futher questions and informations you need</p>
+		<a class="reach-out" href="mailto">rayselementsummit2021@aqua.org</a>
+	</div>
+	<section class="logo-wrapper">
+		<div class="aqua-logo">
+			<p id="logo-p">The Ray's Element Aqua Summit 2021 logo</p>
+		</div>
+		<div class="text">
+			<p class="mobile-text"> The logo of Ray's Aqua summit 2021 was decided through a series of logo competition
+				by stake holders from 8 may to 12 june.</p>
+		</div>
+		<p id="text-logo"> The logo of Ray's Aqua summit 2021 was decided through a series of logo competition
+			by stake holders from 8 may to 12 june. After online public voting and selection commitee's judging
+			process, the logo from photographer Vincent won the price. After consultation with Mr john we have
+			optimized the logo for this year's summit as follow</p>
+		<div class="mobile-image">
+			<img class="logo-pic" src="Assets/Icons/running.jpeg" alt="summit-logo">
+		</div>
+	</section>
+	<hr id="line">
+	<div class="past-summits">
+		<h3>See the past RAY'S Aqua Global Summits</h3>
+		<p>Take a look at the last two summits which took place in Dominincan repulblic and Barbados</p>
+	</div>
+	<section class="Summit-examples">
+		<div class="images">
+			<img src="Assets/Images/barbardos.jpeg" alt="barbados-summit">
+			<img id="d-down" src="Assets/Images/dominican.jpeg" alt="carribean-summit">
+		</div>
+	</section>
+	<section class="partner">
+		<div class="footer-mobile">
+			<h2>Partners</h2>
+			<ul class="partners-mobile">
+				<li><a href="https://www.mozilla.org/en-US/firefox/new/" id="dark">mozilla</a></li>
+				<li><a href="https://www.google.com/">Google</a></li>
+				<li><a href="https://www.naver.com/" id="lightgrey">NAVER</a></li>
+				<li><a href="https://www.crunchbase.com/organization/daum-kakao">daumkakao</a></li>
+				<li><a href="https://www.airbnb.com/">Airbnb</a></li>
+			</ul>
+		</div>
+	</section>
```

Comment:
- [ ] The background colour you have used here is not the same as that of the design template
- [ ] _[Optional]_ You should increase the size of the images as they're too small compared to that of the design template


Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/125109984-f3f03f00-e0db-11eb-9722-f756d7b9f127.png) | ![image](https://user-images.githubusercontent.com/16058470/125110093-0e2a1d00-e0dc-11eb-9a5e-88b7ce8e847b.png)

Reply:
> * [x]  The background colour you have used here is not the same as that of the design template
> * [x]  _[Optional]_ You should increase the size of the images as they're too small compared to that of the design template
> 
> Yours	Original
> ![image](https://user-images.githubusercontent.com/16058470/125109984-f3f03f00-e0db-11eb-9722-f756d7b9f127.png)	![image](https://user-images.githubusercontent.com/16058470/125110093-0e2a1d00-e0dc-11eb-9a5e-88b7ce8e847b.png)

how many minutes would it have taken you to read this thread mate, not more than five minutes. the background color of the image is like that, even with opacity it won't work, unless you want me to be changing the entire picture and using photoshop for it,. this was my last review and you could not even read the thread to follow what other Tse have asked. bloody hell tell me not to get angry, I'm way over it from here onwards.